packages/mobile/src/redux/migrations.ts

```diff
@@ -329,5 +329,14 @@ export const migrations = {
   },
   20: (state: any) => state,
   21: (state: any) => state,
-  22: (state: any) => state,
+  22: (state: any) => ({
+    ...state,
+    i18n: {
+      language: state.app.language,
+      allowOtaTranslations: false,
+      otaTranslationsLastUpdate: 0,
+      otaTranslationsAppVersion: '0',
+      otaTranslationsLanguage: '',
+    },
+  }),
```

Comment:
Can you add a test for this migration?
Also as good practice we should set `state.app.language` to `undefined`.

Reply:
yes i've added this now :) and good shout about the setting language to undefined...i did this for the schema file, but not for this 🙈  my brain .....

as an aside, i was really tempted to do a one liner `expect(migratedSchema).toEqual(v22Schema)` since from the schemas it is already easy to visualise what is changed. but i noticed not many other tests were doing this and i didn't want to look lazy, so i asserted just the changes i was looking for. is there any reason why we don't have a loop test that just goes through every migration and does the `expect(migratedSchema).toEqual(v(n-1)Schema)`? you reckon it'd slow the test run time too much?

Reply:
There's another test which tries all the migrations from the initial schema to be sure they all succeed: https://github.com/valora-inc/wallet/blob/3062f866e458406ca38084c87a0c7153fcc0ea9c/packages/mobile/src/redux/createMigrate.test.ts#L19

Sometimes depending on the migration we need more complex test cases.
But I like what you're proposing! Not sure we've been thorough enough to make it work for all schemas, but feel free to try and open a PR.